tomls/omnibus-base-sepolia-andromeda/spot/statausdc.toml

```
@@ -0,0 +1,72 @@
+# This initializes a sStataUSDC synth per SIP-377 and SCCP-330 for testnet
+# It wraps/unwraps stataUSDC to sStataUSDC 1:1 and buys/sells to sUSD based on the stataUSDC/USDC ratio (because this deployment is denominated in USDC)
+# All exchanges are atomic. The stataUSDC/USDC ratio can be determined via the Erc4626ToAssetsRatioOracle, which calls convertToAssets per ERC-4626
+# On testnet, we use imports.erc_4626_to_assets_ratio_oracle.contracts.ERC20Mock.address as aUSDC and
+# imports.erc_4626_to_assets_ratio_oracle.contracts.ERC4626Mock.address as stataUSDC
+
+[setting.synthStataUsdcMaxMarketCollateral]
+defaultValue = "<%= parseEther(String(5_000_000)) %>"
+
+[setting.spotStrictStalenessTolerance]
+defaultValue = "60"
+
+[invoke.createStataUsdcSynth]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "owner"
+func = "createSynth"
+args = ["Synthetic Static aUSDC", "sStataUSDC", "<%= settings.owner %>"]
+extra.synth_stata_usdc_market_id.event = "SynthRegistered"
+extra.synth_stata_usdc_market_id.arg = 0
+extra.synth_stata_usdc_token_address.event = "SynthRegistered"
+extra.synth_stata_usdc_token_address.arg = 1
+
+[invoke.setSynthPriceStataUsdc]
+target = ["spotFactory.SpotMarketProxy"]
+fromCall.func = "getMarketOwner"
+fromCall.args = ["<%= extras.synth_stata_usdc_market_id %>"]
+func = "updatePriceData"
+args = [
+    "<%= extras.synth_stata_usdc_market_id %>",
+    "<%= extras.erc_4626_to_assets_ratio_oracle_id %>",
+    "<%= extras.erc_4626_to_assets_ratio_oracle_id %>",
+    "<%= settings.spotStrictStalenessTolerance %>",     # Effectively unused, as this oracle node will never return something stale
+]
+
+[invoke.setSynthMaxMarketCollateralStataUsdc]
+target = ["system.CoreProxy"]
+fromCall.func = "owner"
+fromCall.args = []
+func = "configureMaximumMarketCollateral"
+args = [
+    "<%= extras.synth_stata_usdc_market_id %>",
+    "<%= imports.erc_4626_to_assets_ratio_oracle.contracts.ERC4626Mock.address %>",
+    "<%= settings.synthStataUsdcMaxMarketCollateral %>",
+]
+
+[invoke.configureSynthWrapperStataUsdc]
```

nit
very inconsistent naming


For invokes it is much more predictable to use smth like `Contract_funcName_significantArgWhenNeeded`

```suggestion
[invoke.SpotMarketProxy_setWrapper_stataUSDC]
```

Or at least make all of them python_case

---

Done